src/Components/Video/FilterVideos.jsx

```diff
@@ -0,0 +1,54 @@
+import { useState } from "react";
+import { useDataContext } from "../../Context/data-context";
+
+export const FilterVideos = () => {
+  const [searchTxt, setSearchTxt] = useState("");
+  const { dispatch } = useDataContext();
+
+  const searchHandler = (e) => {
+    if (e.keyCode === 13) {
+      dispatch({ type: "SEARCH_VIDEO", payload: searchTxt });
+      setSearchTxt("");
+    }
+  };
```

Comment:
usually, search input will not be cleared after hitting the search because though after hitting the search button, the user might want to edit the search input or add few more keywords